doc/source/qan.rst

```diff
@@ -48,13 +48,12 @@ To start working with |qan|, open the list of dashboards on the |pmm| home
 page. Then, proceed to the |gui.host| field at the top of the page and select a
 host where the |pmm-client| is installed from the list of database instances.
 
-The |qan| dashboard will show summary of the selected host and database activity
-metrics followed with the list of queries in a summary table. Be default, |qan|
-shows the top *ten* queries ranked by :term:`%GTT` (Grand total time) as a
-result of monitoring your database server for the last hour. Each query displays
-three essential metrics: *Load*, *Count*, and *Latency*. Also queries
-which are newly seen within the the time range selected are highlighted with a
-blue color for their better noticeability.
+The |qan| dashboard will show a summary of the selected host and database
+activity metrics followed by the list of queries in a summary table. By
+default, |qan| shows the top *ten* queries ranked by :term:`%GTT` (Grand total
+time). Each query displays three essential metrics: *Load*, *Count*, and
+*Latency*. Also queries which are newly seen within the the time range selected
+are highlighted with a blue color for their better noticeability.
```

Comment:
- change "for their better noticeability" to "for faster identification of new queries"